crates/examples/interactive/README.md

```diff
@@ -4,7 +4,7 @@ This example demonstrates how to use TLSNotary in a simple interactive session b
 
 This example fetches data from a local test server. To start this server, run:
 ```shell
-PORT=4000 cargo run --bin tlsn-server-fixture
+RUST_LOG=info PORT=4000 cargo run --bin tlsn-server-fixture
```

Comment:
why do we need to run this at port 4000 again? i always just use the default port 3000

Reply:
Port 3000 is also the default port for the React development server. So this often conflicts.